tests/cpp/core/PixelStreamWindowManagerTests.cpp

```diff
@@ -271,7 +271,7 @@ BOOST_AUTO_TEST_CASE(hideAndShowWindow)
     windowManager.handleStreamStart(uri);
     ContentWindowPtr window = windowManager.getWindow(uri);
 
-    BOOST_REQUIRE(!window->isHidden());
+    BOOST_REQUIRE(window->isHidden());
     BOOST_REQUIRE(!window->isPanel());
 
     windowManager.hideWindow(uri);
```

Comment:
reverse the order of the tests then, it is not very useful to hide a hidden window